### PR TITLE
Service: Equivalent CO²

### PR DESCRIPTION
**Pull request by pelikhan**
This pull request adds a new service definition for JACDAC.

### CODE REVIEW

File: services/eco2.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Measures equivalent CO² levels.
+
+    identifier: 0x169c9dc6
+    extends: _sensor
+    camel: eCO2
+
+## Registers
+
+     ro e_CO2: u16 ppm { typical_min=400, typical_max=8192 }  @ reading
```
Comment (mmoskal) [inline services/eco2.md:11]: should we do u32? or maybe u22.10? in general, there isn't much advantage using u16/u8 readings for sensors given all the other overheads in the system

Comment (pelikhan) [reply services/eco2.md:11]: use u32

Comment (pelikhan) [reply services/eco2.md:11]: use u22.10

File: services/eco2.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+    const min_e_CO2?: u16 ppm @ min_reading
+
+Minimum measurable value
+
+    const max_e_CO2?: u16 ppm @ max_reading
+
+Minimum measurable value
+
+    const conditioning_period?: u16 min @ 0x180
```
Comment (pelikhan) [inline services/eco2.md:23]: use seconds

Comment (pelikhan) [reply services/eco2.md:23]: u32 it's a const anyway

**Pull request merged**